test/revoke_general_action_test.rb

```diff
@@ -4,17 +4,17 @@
 
   it "returns nil after a successful revoke" do
     actor = User.create!
-    Accessly::PermittedAction.create!(id: SecureRandom.uuid, actor: actor, action: 1, object_type: "Post")
-    _(Accessly::PermittedAction.where(actor: actor).count).must_equal 1
+    Accessly::PermittedAction.create!(actor: actor, action: 1, object_type: "Post")
+    Accessly::PermittedAction.where(actor: actor).count.must_equal 1
```

Comment:
When I run the specs now, I'm getting those deprecation warnings from this that you fixed for us (:tada:) in that other PR. Would we be able to change this back to the `_()` syntax to silence those warnings again? The test output is so nice now thanks to that other PR!